docs/cookbook/recipe_sonata_admin_without_user_bundle.rst

```diff
@@ -0,0 +1,451 @@
+SonataAdmin without FOSUserBundle/SonataUserBundle
+==================================================
+
+Here we will explain how you can use SonataAdmin with Symfony Guard instead
+of using FOSUserBundle or SonataUserBundle.
+
+Code used in this short guide can be found `here <https://github.com/kunicmarko20/guard-talk-example>`__
+with support for different Symfony versions.
+
+.. note::
+    This article assumes you are using Symfony 4.
+
+The recipe
+----------
+
+Things we will need to do:
+
+- User Entity
+- UserProvider
+- LoginForm
+- LoginAuthenticator
+- LoginController
+- Setup firewall in `security.yaml`
+- Login template resembling Sonata style
+
+User Entity
+^^^^^^^^^^^
+
+This represents your security user, you can read more about it `here <http://symfony.com/doc/current/security/entity_provider.html#create-your-user-entity>`__.
+
+.. code-block:: php
+
+    namespace App\Entity;
+
+    use Doctrine\ORM\Mapping as ORM;
+    use Symfony\Component\Security\Core\User\UserInterface;
+
+    /**
+     * @ORM\Entity
+     * @ORM\Table(name="user")
```

Comment:
this is not needed, we should only show the minimal config

Reply:
Table or both annotations?

Reply:
only table